bootstrap.ps1

```diff
@@ -88,13 +89,6 @@ Function Initialize-Proxy {
 }
 
 Function Main {

```

Comment:
There are two Main functions. I find this confusing.

Reply:
Actually, there is only one Main function in bootstrap.ps1. The other one is in a multiline string for the creation of build.ps1.